src/cf-local.ts

```diff
@@ -764,18 +766,25 @@ export async function cfBindLocalServices(
             })
           )
         : []),
+      ...(quoteVcap ? ["-quote-vcap"] : []),
     ],
   });
 }
 
 /**
- * Example of usage : cf bind-local-ups -path .env -service-names serviceNamesValue1 -service-names serviceNamesvalue2 -tags tagsValue1 -tags tagsValue2
+ * Example of usage : cf bind-local-ups -path .env -service-names serviceNamesValue1 -service-names serviceNamesvalue2 -tags tagsValue1 -tags tagsValue2 -quote-vcap
  *
  * @param filePath : string
  * @param instanceNames : string[]
- * @param tags : string[]
+ * @param tags : string[] (Optional)
+ * @param quoteVcap : boolean (Optional)
```

Comment:
please consider to add a meaningful description that reflects in docs

Reply:
done